app/models/miq_provision_virt_workflow.rb

```diff
@@ -377,7 +377,7 @@ def allowed_provision_types(_options = {})
   def allowed_snapshots(_options = {})
     result = {}
     return result if (vm = get_source_vm).blank?
-    vm.snapshots.each { |ss| result[ss.id.to_s] = ss.current? ? "#{ss.name} (Active)" : ss.name }
+    vm&.snapshots&.each { |ss| result[ss.id.to_s] = ss.current? ? "#{ss.name} (Active)" : ss.name }
     result["__CURRENT__"] = _(" Use the snapshot that is active at time of provisioning") unless result.blank?
```

Comment:
won't result be blank for OVF template?

Reply:
This `snapshots` method is only for VM templates. OVF templates don't support this call.